### DISCUSSION TITLE
Allow an 'alg' alias of "RSA256"

### DISCUSSION THREAD

**Discussion by thesiti92** [panva/jose#336] [Ideas]
Right now some JWKS clients (namely `jwks-rsa`) will allow an `alg` value for a JWK to be `RSA256` rather than just `RS256`. For the sake of compatibility with other existing popular libraries, would it be possible to add this as an alias?

**Comment by panva** [comment]
Hi @thesiti92. This is not something I would consider.

**Reply by thesiti92** [reply]
yeahh it was a long shot. thanks for responding so quickly though.
